OJP/OJP_ModesSupport.xsd

```diff
@@ -30,18 +54,24 @@
 	</xs:simpleType>
 	<xs:simpleType name="PrivateModesEnumeration">
 		<xs:annotation>
-			<xs:documentation>[a category of MODE in TMv6] MODEs offered by private individuals </xs:documentation>
+			<xs:documentation>MODE OF OPERATION offered by private individuals (contains the values as defined by NeTEx). In the end these are all different forms of car pooling</xs:documentation>
```

Comment:
so why not PrivateModesOf OperationEnumeration (as we have IndividualModesOfOperationEnumeration)

Reply:
changed in the next commit.